src/panels/lovelace/cards/hui-weather-forecast-card.ts

```diff
@@ -224,13 +225,21 @@ class HuiWeatherForecastCard extends LitElement implements LovelaceCard {
                       ${this.hass!.localize(
                         `ui.card.weather.attributes.${this._config.secondary_info_attribute}`
                       )}
-                      ${stateObj.attributes[
-                        this._config.secondary_info_attribute
-                      ]}
-                      ${getWeatherUnit(
-                        this.hass,
-                        this._config.secondary_info_attribute
-                      )}
+                      ${this._config.secondary_info_attribute !== "wind_speed"
+                        ? html`
+                            ${stateObj.attributes[
+                              this._config.secondary_info_attribute
+                            ]}
+                            ${getWeatherUnit(
+                              this.hass,
+                              this._config.secondary_info_attribute
+                            )}
+                          `
+                        : getWind(
+                            this.hass,
+                            stateObj.attributes.wind_speed,
+                            stateObj.attributes.wind_bearing
+                          )}
```

Comment:
I think this should be reversed. If it is wind speed do something different else case for everything else

Reply:
You are right. Changed